support/approved-access.adoc

```diff
@@ -0,0 +1,31 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="approved-access"]
+= Approved Access
+include::_attributes/common-attributes.adoc[]
+ifdef::openshift-dedicated[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+endif::[]
+:context: approved-access
+
+toc::[]
+
+Red{nbsp}Hat Site Reliability Engineering (SRE) typically does not require access to systems containing customer data as part of normal operations to manage and support {product-title} clusters. In the unlikely event that SRE needs access to systems containing customer data, you can use the _Approved Access_ interface to review and _approve_ or _deny_ access to these systems.
+
+Access requests to customer data on {product-rosa} clusters and the corresponding cloud accounts can be created by SRE either in response to a customer-initiated support ticket or in response to alerts received by SRE as part of the standard incident response process.
+
+When approved access is enabled and an SRE creates an access request, _cluster owners_ receive an email notification informing them of a new access request. The email notification contains a link allowing the cluster owner to quickly approve or deny the access request. You must respond in a timely manner otherwise there is a risk to your SLA for {product-rosa}.
```

Comment:
Here as well:
```suggestion
When Approved Access is enabled and an SRE creates an access request, _cluster owners_ receive an email notification informing them of a new access request. The email notification contains a link allowing the cluster owner to quickly approve or deny the access request. You must respond in a timely manner otherwise there is a risk to your SLA for {product-rosa}.
```